manual/ionwf.tex

```diff
@@ -0,0 +1,71 @@
+\section{Gaussian Product Wavefunction}
+\label{sec:ionwf}
+
+The Gaussian Product wavefunction implements eq.~\ref{eq:gauss_prod_wf}

```

Comment:
Thanks for adding the manual section. Please can you add a citation to any published work using this wavefunction (QMCPACK or not) and a couple of sentences on when/where the wavefunction might be used.

Reply:
@prckent unfortunately I have no knowledge of the history of this wavefunction. The file banner suggests it was first developed by Ken Esler. Could we contact him?

Reply:
I will add a few sentences describing potential use of the wavefunction.